vignettes/getting-started.Rmd

```diff
@@ -60,11 +76,13 @@ example_sample_metadata <- data.frame(
 )
 ```
 
+We'll need to provide `example_counts`, `pg_metadata` and `example_data$idmapping` to `setup_data()`. We can provide `sample_metadata`, but it is not required. (but should it be???)
```

Comment:
I think we can reevaluate whether this should be required as we move on. Trying to envision what a metadata-less analysis might look like. Not sure yet.

Reply:
I'm pretty sure it's going to be necessary looking at the code

Reply:
It's necessary for running exactly as the original code has it, but one of our secondary goals is to make the code generalizeable to other purposes. And the question is can you still calculate genetic interactions without metadata exactly as the original code has it? The answer very well may be no. But I hate to be restrictive too soon.

Reply:
I think this seems like a reasonable start then and then later on we could figure out what additional ways this may need to be adjusted to make it more generalizable.